parts/linux/cloud-init/artifacts/cse_install.sh

```diff
@@ -118,6 +118,22 @@ installCredentalProvider() {
     rm -rf ${CREDENTIAL_PROVIDER_DOWNLOAD_DIR}
 }
 
+# TODO (alburgess) have oras version managed by dependant or Renovate
+installOras() {
```

Comment:
should we also add call to this somewhere in cse_main.sh to ensure it gets installed at runtime if the underlying VHD doesn't already have it cached?

Reply:
The only case where we need oras installed is for a network isolated cluster. So, if it is not cached, it won't be able to be installed on those clusters - since it is needed to install everything else. 

As a general question for my understanding - @Devinwong added the loop inside install_dependencies.sh for installing the packages in components.json. Are any of these packages installed via cse?

Reply:
Some do. For example, in the for-loop for `containerd` case, it calls `installStandaloneContainerd`. And in cse_main.sh around ln124, it also calls `installContainerRuntime` which will eventually calls `installStandaloneContainerd`.
But some don't. Actually recently Paul removed the codes to install cni from cse. So now it will only use what cached in the VHD.

For your case, if oras is not cached as expected, when it comes to network isolated cluster bootstrapping the node, it still won't be able to install oras, right?

Reply:
Right I don't think a NIC will have the ability to install oras